indexd_test_utils2/__init__.py

```diff
@@ -0,0 +1,296 @@
+import hashlib
+import random
+import uuid
+import threading
+import pytest
+import requests
+import flask
+import socket
+
+from pytest_postgresql.janitor import DatabaseJanitor
+
+from indexclient.client import Document, IndexClient
+from indexd import app_init
+from indexd.alias.drivers.alchemy import (
+    Base as alias_base,
+    SQLAlchemyAliasDriver,
+)
+from indexd.auth.drivers.alchemy import SQLAlchemyAuthDriver
+from indexd.index.drivers.alchemy import (
+    Base as index_base,
+    SQLAlchemyIndexDriver,
+)
+from indexd.utils import setup_database
+
+from indexd_test_utils2 import indexd_settings
+
+
+@pytest.fixture(scope="session", autouse=True)
+def pg_url(postgresql_proc):
+    yield f"postgresql://{postgresql_proc.user}:{postgresql_proc.password}@{postgresql_proc.host}:{postgresql_proc.port}/indexd_test"
+
+
+@pytest.fixture(scope='session', autouse=True)
+def setup_indexd_test_database(postgresql_proc):
+    """Set up the database to be used for the tests.
```

Comment:
I might also mention in the docstring that the test data will be deleted on exit. Does DatabaseJanitor only delete the data or does it drop tables too?

Reply:
It drops the database. 
https://github.com/ClearcodeHQ/pytest-postgresql/blob/main/src/pytest_postgresql/janitor.py#L92